pkg/services/dashboards/dashboard_service.go

```diff
@@ -65,6 +68,16 @@ func (dr *dashboardServiceImpl) GetProvisionedDashboardData(name string) ([]*mod
 	return cmd.Result, nil
 }
 
+func (dr *dashboardServiceImpl) GetProvisionedDashboardDataByDashboardId(dashboardId int64) (*models.DashboardProvisioning, error) {
```

Comment:
I hid the command into the service here and using that in the code base.

Reply:
:+1: 
This can be unexported since its only used in this package.

Reply:
It is also used here https://github.com/grafana/grafana/pull/16706/files#diff-21f89324b16a1ac91222160b871178bbR110.